examples/src/utils/MeasurementUi.ts

```diff
@@ -0,0 +1,67 @@
+import { Cognite3DViewer } from "@cognite/reveal";
+import { MeasurementTool } from "@cognite/reveal/tools";
+import dat from "dat.gui";
+
+export class MeasurementUi {
+  private readonly _viewer: Cognite3DViewer;
+  private readonly _measurementTool: MeasurementTool;
+  private _gui: dat.GUI;
+  private _guiController: any[];
+
+  private state = {
+    lineWidth: 0.002,
+    color: 0x00FFFF
+  };
+
+  private pointToPointMeasurement = {
+    pointDistance: false
+  }
```

Comment:
These names dont make that much sense, right? An object containing pointDistance that is a boolean. When I see something called `pointToPointMeasurement` I am expecting measurement data between two points. Also I expect pointDistance to be a number, not a boolean.